python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -78,6 +78,7 @@ class GrapheneAssetNode(graphene.ObjectType):
         beforeTimestampMillis=graphene.String(),
         limit=graphene.Int(),
     )
+    computeKind: graphene.String
```

Comment:
Hey hey! Just a heads up that this wasn't exposed in the GraphQL schema and I am changing it to this in a follow up PR

`computeKind = graphene.String()`